features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/developer/tracing/ConfigureTracingStateProvider.kt

```diff
@@ -21,7 +21,7 @@ open class ConfigureTracingStateProvider : PreviewParameterProvider<ConfigureTra
 
 fun aConfigureTracingState() = ConfigureTracingState(
     targetsToLogLevel = persistentMapOf(
-        Target.COMMON to LogLevel.INFO,
+        Target.ELEMENT to LogLevel.INFO,
```

Comment:
Note to reviewer: this is just updating a state used for the preview (and the screenshot test).